app/uk/gov/hmrc/tradergoodsprofilesdatastore/controllers/GetRecordsController.scala

```diff
@@ -63,20 +62,8 @@ class GetRecordsController @Inject() (
   def getRecordsCount(
     eori: String
   ): Action[AnyContent] = identify.async { implicit request =>
-    routerConnector.getRecords(eori, page = Some(0), size = Some(1)).map { recordsResponse =>
+    routerConnector.getRecords(eori, page = Some(recursiveStartingPage), size = Some(1)).map { recordsResponse =>
```

Comment:
page 0 here no? To get the first page

Reply:
the size is the pagesize not number, it should be one because we want a tiny page as we are only getting the total records

Reply:
Agreed. Brain fart.